.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
```

Comment:
@rpappalax I don't know nothing about nothing.
But if I remove that `E501`, then I get aboot _53_ flake8 errors.